cluster/ci/config/prow/openshift/tracer.yaml

```diff
@@ -42,13 +48,45 @@ objects:
       spec:
         serviceAccountName: "${NAME}"
         containers:
+        - name: oauth-proxy
+          image: openshift/oauth-proxy:v1.0.0
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 8443
+            name: web
+          args:
+          - -provider=openshift
+          - -https-address=:8443
+          - -http-address=
+          - -email-domain=*
+          - -upstream=http://localhost:8080
+          - -client-id=system:serviceaccount:ci:${NAME}
+          - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          - '-openshift-sar={"resource": "clusterroles", "verb": "get"}'
+          - '-openshift-delegate-urls={"/": {"resource": "clusterroles", "verb": "get"}}'
```

Comment:
@smarterclayton I expected the proxy to pick up what I can perform in the cluster and allow me to access what it's proxying to. Instead:
```
2018/01/05 15:38:03 oauthproxy.go:657: 172.16.6.1:60776 Cookie "_oauth_proxy" not present
2018/01/05 15:38:03 provider.go:347: authorizer reason: User "system:anonymous" cannot get clusterroles at the cluster scope
```

Reply:
https://tracer-ci.svc.ci.openshift.org/

Reply:
OK, I've got it figured out. I was missing the oauth reference annotation on the service account.